apps/admin-ui/src/clients/keys/ImportKeyDialog.tsx

```diff
@@ -53,6 +46,15 @@ export const ImportKeyDialog = ({
     name: "keystoreFormat",
     defaultValue: "JKS",
   });
+
+  const baseFormats = useServerInfo().cryptoInfo?.supportedKeystoreTypes;
```

Comment:
Providing a default value here will remove the need to use optional chaining later on.
```suggestion
  const baseFormats = useServerInfo().cryptoInfo?.supportedKeystoreTypes ?? [];
```